plugins/modules/system/disk_facts.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+#
+# Copyright 2021 VMware, Inc
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: disk_facts
+version_added: "2.5.0"
+short_description: Get disk facts in the linux OS
+description:
+    - This module is to get disk facts like free space, used percentage, mount location etc in linux Ansible controller/controlled machines.
+requirements:
+  - psutil(python module)
+options:
+    name:
+        description:
+            - This is the disk name to query
+        required: false
+        default: all
+        type: str
+    filter:
+        description:
+            - This is the parameter used to filter the output
+        required: false
+        type: list
+        elements: str
+author:
+    - Saranya Sridharan(@saranyasridharan), Swetha M, Nagendra Kini, Anup Kumar, Mehmoodkha Pathan
```

Comment:
```suggestion
    - Saranya Sridharan (@saranyasridharan)
    - Swetha M
    - Nagendra Kini
    - Anup Kumar
    - Mehmoodkha Pathan
```
You also need to specify GitHub handles for the others, or add `(!UNKNOWN)` instead of `(@githubhandle)`.